CONTRIBUTING.md

```diff
@@ -524,6 +490,39 @@ into any of those categories, use your best judgement.
 * Use `HasX` for protocols that have readable and/or writable attributes
   or getter/setter methods (e.g. `HasItems`, `HasFileno`).
 
+### Incomplete stubs
+
+We accept partially annotated stubs, especially for larger packages.
+These need to follow the following guidelines:
+
+* A package must include all public submodules, all public classes,
+  functions and methods, and all other public fields.
+* Included functions and methods must list all arguments, but the argument
```

Comment:
I imagine these rules could seem arbitrary and complicated to someone new to typeshed. We should mention that the `create_baseline_stubs` script generates stubs that follow these rules.

Reply:
@Akuli What do you think about the proposal here: https://github.com/python/typeshed/pull/9548#issuecomment-1386880372 This is much simpler.

Reply:
I like it, it is less confusing :)